packages/react-examples/src/react/OverflowSet/OverflowSet.Vertical.Example.tsx

```diff
@@ -16,6 +16,7 @@ const onRenderItem = (item: IOverflowSetItemProps): JSX.Element => {
   return (
     <CommandBarButton
       role="menuitem"
+      title={item.title}
```

Comment:
It's probably a better idea to wrap the `CommandBarButton` with a `TooltipHost` instead of using the html `title` attribute for consistency in the website.

Reply:
Very good point, will update!